.travis.yml

```diff
@@ -45,10 +45,11 @@ script:
   - which mlflow
   - echo $MLFLOW_HOME
   - mlflow sagemaker build-and-push-container --no-push --mlflow-home .
-  - pytest --cov=mlflow --verbose --large
   - ./lint.sh
+  - pytest --cov=mlflow --verbose --large
```

Comment:
I moved the Python lint earlier here to make it fail faster

Reply:
Do you mean to run `lint.sh` before and after this?

Reply:
Ahh, one is cd'd. I see.